engine/src/main/java/com/redhat/cloud/notifications/recipients/rbac/RbacRecipientUsersProvider.java

```diff
@@ -82,10 +83,25 @@ public void initCounters() {
     @CacheResult(cacheName = "rbac-recipient-users-provider-get-users")
     public List<User> getUsers(String accountId, boolean adminsOnly) {
         Timer.Sample getUsersTotalTimer = Timer.start(meterRegistry);
+
         List<User> users;
         if (retrieveUsersFromIt) {
-            final List<ITUserResponse> itUserResponses = itUserService.getUsers(accountId, adminsOnly);
-            users = transformToUser(accountId, itUserResponses, adminsOnly);
+            List<ITUserResponse> usersPaging;
+            List<ITUserResponse> usersTotal = new LinkedList<>();
+
+            int pagingStart = 0;
+            int pagingEnd = 10000;
```

Comment:
I wonder if making this value configurable could be useful to fine-tune the value.

Reply:
hm. we can. i don't think we definitely need to do this as we will have to do a deployment each time we change it anyway. 
as long as everyone of us is able to understand the code, I would also be fine with this solution, but also with a constant or something else. Don't have a strong opinion here

Reply:
Deploying a configuration change has a much lower impact than deploying a new version of the app.